webpush/push.go

```diff
@@ -69,6 +69,25 @@ func NewPushRequest(sub *Subscription, message string, token string) (*http.Requ
 	return req, nil
 }
 
+// NewVapidRequest creates a valid Web Push HTTP request for sending a message
+// to a subscriber, using Vapid authentication. You can add more headers to
+// configure collapsing, TTL.
+func NewVapidRequest(sub *Subscription, message string, vapid *Vapid) (*http.Request, error) {
```

Comment:
Would be really nice if I, as a developer, could understand when to use NewPushRequest vs NewVapidRequest.
Also, link to the draft maybe? Would be useful, too.